scripts/us_cdc/500_places/README.md

```diff
@@ -0,0 +1,94 @@
+
+### Notes and Caveats
+
+None.
+
+### License
+The data is made available for public-use by the [CDC](https://www.cdc.gov/nchs/data_access/ftp_data.htm). Users of CDC National Center for Health Statistics Data must comply with the CDC's [data use agreement](https://www.cdc.gov/nchs/data_access/restrictions.htm).
+
+### Dataset Documentation and Relevant Links
+These data were collected and provided by the [CDC National Center for Chronic Disease Prevention and Health Promotion](https://www.cdc.gov/chronicdisease/index.htm). The documentation for the datasets is accessible [here](https://www.cdc.gov/places/about/index.html).
+
+## About the Import
+
+### Artifacts
+
+#### Scripts
+[`parse_cdc_places.py`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/parse_cdc_places.py)
+
+#### Test Scripts
+[`parse_cdc_places_test.py`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/parse_cdc_places_test.py)
+
+#### Test Files
+[`sample_county.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/test_data/sample_county.csv)
+
+[`sample_county_expected.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/test_data/sample_county_expected.csv)
+
+[`sample_city.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/test_data/sample_city.csv)
+
+[`sample_city_expected.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/test_data/sample_city_expected.csv)
+
+[`sample_census_tract.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/test_data/sample_census_tract.csv)
+
+[`sample_census_tract_expected.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/test_data/sample_census_tract_expected.csv)
+
+[`sample_zip_code.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/test_data/sample_zip_code.csv)
+
+[`sample_zip_code_expected.csv`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/test_data/sample_zip_code_expected.csv)
+
+#### tMCFs
+[`cdc_places.tmcf`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/cdc_places.tmcf)
+
+### Import Procedure
+
+#### Testing
+
+##### Test Data Cleaning Script
+
+To test the data cleaning script, run:
+
+```bash
+$ python3 parse_cdc_places_test.py
+```
+
+The expected output of this test can be found in the [`test_data`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/500_places/test_data/) directory.
```

Comment:
We should include a __init__.py in this directory so that the test here can be run as part of the pre-submit tests.

Note that though once you add it you might need to refer to the main module as

`from .parse_cdc_places ...`

and run the script as:

`python3 -m unittest discover -v -s scripts/us_cdc -p *_test.py`

Reply:
done